components/Nav/NavListItem/__tests__/NavListItem.test.js

```diff
@@ -94,4 +94,26 @@ describe('NavListItem', () => {
 
     expect(wrapper.find('ul')).toHaveClassName('invisible');
   });
+
+  it('should hide sublinks on pressing Shift+Tab on first sublink', () => {
+    const wrapper = shallow(<NavListItem {...testDataWithSublinks} />);
+
+    wrapper.find('button').simulate('click');
+    wrapper
+      .find('[data-testid="Nav Item Test - 1"]')
+      .simulate('keyDown', { key: 'Tab', shiftKey: true });
+
+    expect(wrapper.find('ul')).toHaveClassName('invisible');
+  });
+
+  it('should hide sublinks on pressing Tab on last sublink', () => {
+    const wrapper = shallow(<NavListItem {...testDataWithSublinks} />);
+
+    wrapper.find('button').simulate('click');
```

Comment:
Can you add an assertion that shows the the sub links are
visible here (in both tests)?

That was we know the whole user story is functioning properly.

Reply:
I considered adding them but I was not sure if I should do that or not. Just give me a minute I would fix this.

Reply:
Really stellar tests for somebody who is unfamiliar. You're killing it!